### PR TITLE
Landing and Contact Page

### PR DESCRIPTION
**Pull request by brandenbyers**
I haven't spent any more time with this since I first showed you the draft a couple of months ago. But I would like to get this up so that we can show Nina at LCM (I told her to expect a link to our new website when I talked to her on the phone). Obviously, we want to add events and other such details soon, but I think it better to get something more than the seriously limited landing page that we currently have.

Any objections or last minute corrections?

EDIT: If either of you want to run this locally before approving, you can do so by simply `gem install jekyll` (assuming you have Ruby installed) and then running `jekyll serve`.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,7 +1,97 @@
...
+    </ul>
+  </div>
+</section>
+
+<!-- Two -->
+<section id="two" class="wrapper alt style2">
+  <section class="spotlight">
+    <div class="image"><img src="/images/meetup-sign.jpg" alt="" /></div><div class="content">
+      <h2>We bring keyboard <br/>people together.</h2>
+      <p>And we bring people together through keyboards. We want you to join us at our next meetup or workshop whether it will be your first time touching a mechanical keyboard or you've been doing the mechanical since before it was hip.</p>
```
Comment (jetpacktuxedo) [inline index.html]: "been doing the mechanical"?

I'm sure there is a better way to phrase that, but I'm not really sure how.

Comment (JordonPhillips) [inline index.html]: "been clicking and clacking"

Comment (brandenbyers) [inline index.html]: Updated. Anything else?

**Comment by jetpacktuxedo** [comment]
lgtm, I'd say give Jordon a day to check it out, and if he doesn't say anything merge it

**Pull request merged**